app/services/foreman/renderer/scope/macros/helpers.rb

```diff
@@ -171,7 +171,9 @@ def generate_web_request(utility:, url:, ssl_ca_cert: nil, headers: nil, params:
             utility = Foreman.download_utilities.fetch(utility || 'curl')
             command = ["#{utility[:download_command]} #{url}"]
             command << "#{utility[:ca_cert]} #{ssl_ca_cert}" if ssl_ca_cert
+            command << "--fail" if utility[:fail]
```

Comment:
```suggestion
            command << utility[:fail] if utility[:fail]
```
As `utility[:fail]` is already `--fail` for curl (and `nil` for wget)